hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -71,42 +107,50 @@ public int getZkfcHeapSize() {
   }
 
   public int getTaskHeapSize(String taskName) {
+    int size = DEFAULT_HEAP_SIZE;
```

Comment:
returning via cases added to the complexity metric which we are trying to reduce.    I'll init to 0.